src/config.py

```diff
@@ -0,0 +1,15 @@
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+INFURA_KEY = os.getenv('INFURA_KEY')
```

Comment:
Since we will not use infura when we deploy this, i would suggest that you have some generic rpc variables, one per chain, that you read from the environment, and then in your local set up you can hardcode the infura and the gnosis rpc you are using.

I.e., let's not make it look like one has to necessarily use infura